computer-science/index.json

```diff
@@ -1,5 +1,11 @@
 {
     "resources": [
+        {
+            "title": "Teach Yourself Computer Science",
+            "url": "https://teachyourselfcs.com/",
+            "type": "website",
+            "level": "everyone"
```

Comment:
hey, thanks for this. Can you maybe add a review on how did you like this resource?